tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -402,12 +402,9 @@ $(TFL_CC_HDRS)
 
 # TODO(b/165940489): Figure out how to avoid including fixed point
 # platform-specific headers.
+# some kiss fft source file has to be included in header sections because
+# the implemenation of three different resolution fft from one single c file.
```

Comment:
perhaps add the following to the comment:

See http://b/201319430 for additional context

Reply:
Done